### PR TITLE
[Frictionless] Limit Allowed Upload Types

### PR DESCRIPTION
**Pull request by JingleH**
As of now, no constraints exist on the SEO side for upload file types. After this PR, you should see that uploading pngs to a jpg->png QA would show an error toast message.

Resolves: https://jira.corp.adobe.com/browse/MWPW-158184

Test steps:

1. Go to a Frictionless QA page below
2. Drag and drop a file that doesn't have the right file type
3. A red error toast message should appear

Test URLs:
- After: https://frictionless-image-types--express--adobecom.hlx.live/express/feature/image/convert/png-to-jpg?martech=off Should not accept png
- https://frictionless-image-types--express--adobecom.hlx.live/express/feature/image/convert/jpg-to-png?martech=off Should not accept jpg
- https://frictionless-image-types--express--adobecom.hlx.live/express/feature/image/remove-background?martech=off should still accept all

### CODE REVIEW

**Review by echen-adobe — COMMENTED** [review]
Will this be editable by authoring? This will be confusing on the pages that only accept a single file type.

Screenshot 2024-09-13 at 1 50 37 PM

**Comment by JingleH** [comment]
> Will this be editable by authoring? This will be confusing on the pages that only accept a single file type.
> 
> Screenshot 2024-09-13 at 1 50 37 PM

@echen-adobe yes we will have separate authoring tickets to update those sentences in word docs

**Review by echen-adobe — APPROVED** [review]

**Review by kennethlum — APPROVED** [review]

File: express/blocks/frictionless-quick-action/frictionless-quick-action.js [hunk 1]
```diff
@@ -42,206 +73,194 @@ function selectElementByTagPrefix(p) {
...
+    close.addEventListener('click', () => {
+      toast.classList.add('hide');
+    });
+    toast.append(close);
+    block.append(toast);
+  }
+  toast.classList.remove('hide');
+  setTimeout(() => {
+    toast.classList.add('hide');
+  }, 4500);
```
Comment (qiyundai) [inline express/blocks/frictionless-quick-action/frictionless-quick-action.js:252]: Not sure if is relevant -- I personally think it's fine but might be worth noting that there's some accessibility compliances that Spectrum abides by that says toast message automatic dismiss delay should be minimum 6 seconds.

Comment (JingleH) [reply express/blocks/frictionless-quick-action/frictionless-quick-action.js]: updated to 6 sec it is!

**Pull request merged**